client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -54,20 +68,36 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 	};
 
 	renderLocalBusinessCard() {
-		const { siteSlug, translate } = this.props;
-
-		let buttonProps = {};
-
-		if ( config.isEnabled( 'google-my-business' ) ) {
-			buttonProps = {
-				buttonHref: `/google-my-business/new/${ siteSlug }`,
-			};
+		const { canUserManageOptions, translate } = this.props;
+
+		let connectButton;
+
+		if ( config.isEnabled( 'google-my-business' ) && canUserManageOptions ) {
+			connectButton = (
+				<KeyringConnectButton
+					serviceId="google_my_business"
+					onClick={ this.trackCreateMyListingClick }
+					onConnect={ this.handleConnect }
+				>
+					{ translate( 'Create Your Listing', {
+						comment: 'Call to Action to add a business listing to Google My Business',
+					} ) }
+				</KeyringConnectButton>
+			);
 		} else {
-			buttonProps = {
-				buttonHref: 'https://www.google.com/business/',
-				buttonIcon: 'external',
-				buttonTarget: '_blank',
-			};
+			connectButton = (
+				<Button
+					primary
+					href="https://www.google.com/business/"
```

Comment:
Please pay attention during rebase, this url has changed.

Reply:
Thanks! Saved me! =)